### PR TITLE
Refactor initialize method to remove activesupport

### PR DESCRIPTION
**Pull request by adeluccar**
Just wanted to check if it was possible to get all the functionality out of the gem without the activesupport requirements. It's perfectly functional and returns everything as detailed in the gem's README.
Maybe consider a pull request? :+1:

### CODE REVIEW

File: lib/21-day-challenge-countdown.rb [hunk 1]
```diff
@@ -9,21 +6,21 @@ class Countdown
...
+      next_deadline = start_time + current_day_in_seconds - 1 # one second is substracted at the end of day
+      left_today = next_deadline - now
+      @time_left = Time.at(left_today).gmtime
       # left_string = @time_left.strftime("%H:%M:%S")
     end
 
     def deadline
-      "Today is day #{current_day}. You have #{time_left.strftime('%H:%M:%S')
-                                             } left to send your pull request."
+      "Today is day #{current_day}. You have #{time_left.strftime('%H')} hours, #{time_left.strftime('%M')} minutes and #{time_left.strftime('%S')} seconds left to send your pull request."
```
Comment (kotp) [inline lib/21-day-challenge-countdown.rb]: Keep the lines at 80 columns if possible.

Tests run unaffected?

Comment (adeluccar) [inline lib/21-day-challenge-countdown.rb]: @kotp What tests? I don't see any specs. Unless you mean 'hands testing by Alberto' in which case, yes. All returns I tested are the same as the gem's.

Re: .deadline's return, how could I accomplish that? It's a particular long string due to string interpolation.

**Comment by adeluccar** [comment]
Hmm. If I break the string up, it will be rendered with the format columns in the command line too.
Any suggestions on how to make this happen? I mean, keep the 80 column limit but still receive the whole string in one line? @kotp

**Comment by adeluccar** [comment]
Made a few mistakes rebasing, so I'm going to close this branch and start a new one.

**Pull request closed without merging**